core-web/libs/sdk/experiments/src/lib/parser/parser.ts

```diff
@@ -82,71 +81,49 @@ const verifyRegex = (regexToCheck: string | null, href: string): boolean => {
  * There could be scenarios where none of these conditions are met, in that case, dataToStorage will be the default empty object.
  */
 export const parseData = (
-    fetchExperiments: AssignedExperiments | null,
-    storedExperiments: AssignedExperiments | null
-): AssignedExperiments => {
-    let dataToStorage: AssignedExperiments = {} as AssignedExperiments;
+    fetchExperiments: Experiment[] | undefined,
+    storedExperiments: Experiment[] | undefined
+): Experiment[] => {
+    let dataToStorage: Experiment[] = {} as Experiment[];
 
     if (fetchExperiments && !storedExperiments) {
         // TODO: Use fetchExperiment instead fetchExperimentsNoNoneExperimentID when the endpoint dont retrieve NONE experiment
         // https://github.com/dotCMS/core/issues/27905
-        const fetchExperimentsNoNoneExperimentID: AssignedExperiments = {
-            ...fetchExperiments,
-            experiments: fetchExperiments?.experiments
-                ? fetchExperiments.experiments.filter((experiment) => experiment.id !== 'NONE')
-                : []
-        };
-
-        dataToStorage = {
-            ...fetchExperimentsNoNoneExperimentID,
-            experiments: [
-                ...setExpireTimeNewExperiments(fetchExperimentsNoNoneExperimentID.experiments)
-            ]
-        };
+        const fetchExperimentsNoNoneExperimentID: Experiment[] = fetchExperiments
+            ? fetchExperiments.filter((experiment) => experiment.id !== 'NONE')
+            : [];
+
+        dataToStorage = setExpireTimeNewExperiments(fetchExperimentsNoNoneExperimentID);
```

Comment:
```suggestion
        dataToStorage = getExpireTimeNewExperiments(fetchExperimentsNoNoneExperimentID);
```

Reply:
In this case, I think need to be `set`, because the endpoint give to me a lookbackWindow like this:
```
"lookBackWindow": {
    "expireMillis": 1209600000,
    "value": "FEaiWEQQFv1gyyDpDmZM"
},
```
And I added `expireTime`
```
"lookBackWindow": {
     "expireMillis": 1209600000, // 14 days
     "value": "FEaiWEQQFv1gyyDpDmZM",
      "expireTime": Date.now + expireMillis //With this I will remove this experiment from the browser in 14 days.
},
```
maybe could be `addExpireTimeToExperiment`

Reply:
I changed to `addExpireTimeToExperiment`.